src/main/python/hydra/lib/hydrabase.py

```diff
@@ -752,6 +766,20 @@ def delete_all_launched_apps(self, timeout=12, threshold=200, instance_batch=100
         for app in self.app_id_list:
             self.__mt.wait_app_removal(app)
 
+    def delete_all_app_instances_groups(self):
+        """
+        Delete all existing hydra app instances groups
+        :return: 0 in case of success. 1 otherwise.
+        """
+        l.info("Deleting all groups")
+        for group_name in self.app_group.keys():
+            if self.delete_app_instances_group(group_name):
```

Comment:
see this becomes confusing now, usually
if something:
    do_something

Now its the reverse, probably coming from bash :)